src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/response/ScheduleItem.kt

```diff
@@ -86,6 +87,18 @@ data class ScheduleItem(
   )
   val prisonLocKey: String,
 
+  @Schema(
+    description = "The location description where the appointment takes place. The localName from the locations-inside-prison service).",
+    example = "VCC-crown-conference-room-1",
+  )
+  val prisonLocDesc: String?,
```

Comment:
Will `prisonLocDesc` and `dpsLocationId` ever be null?

Reply:
Nope. Made both mandatory.